bpytop.py

```diff
@@ -3575,7 +3585,11 @@ def create_tree(pid: int, tree: defaultdict, indent: str = "", inindent: str = "
 						det_cpu = getinfo["cpu_percent"]
 				if "username" in getinfo and isinstance(getinfo["username"], float): getinfo["username"] = ""
 				if "cmdline" in getinfo and isinstance(getinfo["cmdline"], float): getinfo["cmdline"] = ""
-				for value in [ name.lower(), str(pid), getinfo.get("username", "").lower(), " ".join(getinfo.get("cmdline", "")).lower() ]:
+				if cls.case_sensitive:
+					process_info = [ name, str(pid), getinfo.get("username", ""), " ".join(getinfo.get("cmdline", "")) ]
+				else:
+					process_info = [ name.lower(), str(pid), getinfo.get("username", "").lower(), " ".join(getinfo.get("cmdline", "")).lower() ]
+				for value in process_info:
```

Comment:
Would this work instead:
line 3593 `if cls.case_sensitive: value = value.lower()`?
Would remove the need for the if, else block above.